src/main/java/com/whalespottingjava/controllers/SightingController.java

```diff
@@ -65,4 +65,11 @@ public String getAllSightings(Model model) {
   @GetMapping("/sightings/approved")
   @ResponseStatus(HttpStatus.I_AM_A_TEAPOT)
   public void getAllApprovedSightings() {}
+
+  @GetMapping("/sightings/pending")
+  @ResponseStatus(HttpStatus.OK)
+  public String getPendingSightings(Model model) {
+    model.addAttribute("sightings", sightingService.getAllPendingSightings());
+    return "sightings_map";
```

Comment:
I would add a line before return for style purpose and clear code

Reply:
I've added the blank line now

Reply:
OK, I understand how it works from JAVA/Spring. Should it be display on the admin page then? As it is the admin who wants to see all the pending?